### PR TITLE
Feature/signup form

### PR DESCRIPTION
**Pull request by AnastasiaLes**
Fixed: 
1. all colors are in a separate file
To check:
1. Sign Up Form
2. Navigation between Login and Sign up pages
3. Collect data from forms

Screenshot 2022-11-09 205500

### CODE REVIEW

File: apps/freelance/src/redux/auth/auth-slice.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+};
+
+const userSlice = createSlice({
+    name: 'user',
+    initialState,
+    reducers: {
+        setUser(state, action) {
+            state.email = action.payload.email;
+            state.token = action.payload.token;
+            state.id = action.payload.id;
```
Comment (lyudad) [inline apps/freelance/src/redux/auth/auth-slice.ts:16]: no id should be stored, its not secure

File: libs/components/src/lib/login-form/login-form.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import * as yup from 'yup';
+import {yupResolver} from '@hookform/resolvers/yup';
```
Comment (lyudad) [inline libs/components/src/lib/login-form/login-form.tsx:3]: some rules are missed it should be `import { yupResolver } ...`

File: apps/freelance/src/redux/auth/auth-slice.ts (hunk 1 above)
Comment (AnastasiaLes) [reply apps/freelance/src/redux/auth/auth-slice.ts:16]: fixed

File: apps/freelance/src/pages/SignupPage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+
+import { SignUpForm } from '@freelance/components';
+import { Wrapper} from './styles';
+import { Button } from 'antd';
```
Comment (lyudad) [inline apps/freelance/src/pages/SignupPage/index.tsx:4]: import rules:
node modules
absolute path
relative path

File: apps/freelance/src/pages/SignupPage/styles.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+margin-top: 50px;
+margin-left: 100px;
+`;
```
Comment (lyudad) [inline apps/freelance/src/pages/SignupPage/styles.ts:6]: something with eslint
```
export const Wrapper = styled.div`
    margin-top: 50px;
    margin-left: 100px;
`;
```
it should look like it

File: libs/components/src/lib/login-form/login-form.tsx (hunk 2 above)
Comment (AnastasiaLes) [reply libs/components/src/lib/login-form/login-form.tsx:3]: rules were added

File: apps/freelance/src/pages/SignupPage/index.tsx (hunk 3 above)
Comment (AnastasiaLes) [reply apps/freelance/src/pages/SignupPage/index.tsx:4]: fixed 
image

File: apps/freelance/src/pages/SignupPage/styles.ts (hunk 4 above)
Comment (AnastasiaLes) [reply apps/freelance/src/pages/SignupPage/styles.ts:6]: checked

**Pull request merged**